site/data/dynamo.ts

```diff
@@ -33,3 +34,19 @@ export const getDisruptionsDataFromDynamo = async () => {
         return parsedItem.data;
     }).filter(notEmpty);
 };
+
+export const insertPublishedDisruptionIntoDynamo = async (disruption: PtSituationElement) => {
+    logger.info("Inserting disruptions data into DynamoDB table...");
```

Comment:
Trying to think if anything would be useful to log out here, I don't know if the SituationNumber would be helpful when reading the logs as we wont know it would we?